doc/user/sql/create-sink.md

```diff
@@ -26,22 +26,28 @@ Field | Use
 **IF NOT EXISTS** | If specified, _do not_ generate an error if a sink of the same name already exists. <br/><br/>If _not_ specified, throw an error if a sink of the same name already exists. _(Default)_
 _sink&lowbar;name_ | A name for the sink. This name is only used within Materialize.
 _item&lowbar;name_ | The name of the source or view you want to send to the sink.
+**FILE** _path-prefix_ | The absolute path and file name prefix of file to create and write to.
+
```

Comment:
Can we add a line for **AVRO OCF** here detailing what that option does?

Reply:
So I tried to rephrase the summary for path_prefix in a way that would make sense. Let me know if it actually does

Reply:
The term prefix is throwing me off here because it's an entire filename; it's not strictly a prefix. I think the term `path_prefix` here could be fine, though, if we're more verbose here. Something like:

> The absolute path and filename you want to create and write to. The filename will be modified to let Materialize create a unique file each time Materialize starts, but the file extension will not be modified. You can find more details !MAKE THIS A LINK! here !END LINK!

I'm not saying you must use that verbatim, but something akin to that would help someone understand what to expect based on what they enter, I think.

Reply:
Ok that makes sense I'll update